mlflow/deployments/utils.py

```diff
@@ -15,3 +19,44 @@ def parse_target_uri(target_uri):
             + "Deployment URIs must be of the form 'target' or 'target:/suffix'"
         )
     return parsed.scheme
+
+
+def _is_valid_uri(uri: str) -> bool:
+    """
+    Evaluates the basic structure of a provided gateway uri to determine if the scheme and
```

Comment:
Can we remove "gateway" from the docstrings / method names?

Reply:
Removed